src/zkaf/apps/src/near.rs

```diff
@@ -144,3 +147,46 @@ pub async fn verify_near_proof(journal_output: Vec<u8>, token_metadata: TokenMet
 
     Ok(res.unwrap())
 }
+
+
+
+/// generate the nft payload
+pub fn generate_tweet_nft_payload(
```

Comment:
I feel as though there is duplicate logic between `zkaf` and `near_indexer`. 
It may be wiser to merge these into an `orchestrator` folder, isolating indexing related logic to a file `near_indexer.rs`, etc.

In essence, this is a part of tidying up the repo.